package.json

```diff
@@ -67,6 +67,7 @@
     "exceljs": "~3.5.0",
     "googleapis": "^61.0.0",
     "jszip": "^3.5.0",
+    "svelte-icons": "^2.1.0",
```

Comment:
Good idea 👍

Reply:
Haha, I'm gonna veto this!

I'd prefer _not_ to include a library in the `app` bundle to add a _single_ icon. I'd have to measure how much larger the app bundle gets by adding this library. This is definitely a user experience vs. developer experience trade-off, and in terms of frontend code, I usually favour user experience, even if it makes things more annoying or complicated for us devs.

In general, adding dependencies to the `worker` bundle is fine, but adding things to the `app` I'm a lot more cautious about.

Reply:
Okay, I did the calculations and it only adds 802 bytes, so it's fine haha! 😅